ui/src/routes/devices.$id.settings.tsx

```diff
@@ -161,7 +161,7 @@ export default function SettingsRoute() {
                     to="keyboard"
                     className={({ isActive }) => (isActive ? "active" : "")}
                   >
-                    <div className="flex items-center gap-x-2 rounded-md px-2.5 py-2.5 text-sm transition-colors hover:bg-slate-100 dark:hover:bg-slate-700 [.active_&]:bg-blue-50 [.active_&]:!text-blue-700 md:[.active_&]:bg-transparent dark:[.active_&]:bg-blue-900 dark:[.active_&]:!text-blue-200 dark:md:[.active_&]:bg-transparent">
+                    <div className="flex items-center gap-x-2 rounded-md px-2.5 py-2.5 text-sm transition-colors hover:bg-slate-100 dark:hover:bg-slate-700 in-[.active]:bg-blue-50 in-[.active]:text-blue-700! md:in-[.active]:bg-transparent dark:in-[.active]:bg-blue-900 dark:in-[.active]:text-blue-200! dark:md:in-[.active]:bg-transparent">
```

Comment:
This is because of the new Tailwind version... since this code didn't exist in _dev_ when I did the upgrade, it needs to be adjusted. I just duplicated the styles on the [Mouse div](https://github.com/dlorch/kvm/pull/2/files#diff-77c0839074210b9099a671eb620f3fdcd5a67e86b180c741c98aefee33247d28R152)